src/serac/infrastructure/output.cpp

```diff
@@ -18,6 +18,29 @@
 
 namespace serac::output {
 
+void outputCurves(const axom::sidre::DataStore& datastore, const std::string& data_collection_name,
+                  const Language language)
+{
+  auto [_, rank] = getMPIInfo();
+  if (rank != 0) {
+    return;
+  }
+
+  std::string output_language = "";
+  if (language == Language::JSON) {
+    output_language = "json";
+  } else if (language == Language::HDF5) {
+    // FIXME: remove this error when implemented, do we even want hdf5 curves?
+    SLIC_ERROR_ROOT("hdf5 has not been implemented in outputCurves");
+    output_language = "hdf5";
+  } else if (language == Language::YAML) {
+    output_language = "yaml";
+  }
+
+  const std::string file_name = fmt::format("{0}_curves.{1}", data_collection_name, output_language);
+  datastore.getRoot()->getGroup("serac_curves")->save(file_name, output_language);
```

Comment:
hmm, I don't really think of JSON or HDF5 as "languages", is this `Language` enum defined in serac or somewhere out of our control?

Reply:
That enum is ours (we have a couple that slightly overlap), would you feel better out `Format`?  They are lumped in under language (in the most broadest sense of the CS word) in many places, well at least JSON/YAML.

Reply:
For this particular function, it seems like `FileExtension`, `FileFormat` or similar would be natural names, but I don't know if those would make sense for other places the enum gets used.

Reply:
I'll review and i think `FileFormat` will work in all places.  I will swap it shortly